src/app/GitUI/CommandsDialogs/BrowseDialog/FormUpdates.cs

```diff
@@ -145,7 +145,9 @@ private void Done()
 
                 if (!Visible)
                 {
-                    ShowDialog(OwnerWindow);
+#pragma warning disable WFO5002 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
```

Comment:
[nit] This and other experimental API warnings can be suppressed in Directory.Build.props